docs/content/dagster-cloud/getting-started/getting-started-with-hybrid-deployment.mdx

```diff
@@ -0,0 +1,120 @@
+description: "Get up and running with Dagster Cloud using a Hybrid Deployment."
+
+platform_type: "cloud"
+---
+
+# Getting Started with Dagster Cloud using a Hybrid Deployment
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Welcome to Dagster Cloud!
+
+In this guide, we'll give you everything you need to get up and running with Dagster Cloud using a [Hybrid Deployment](/dagster-cloud/deployment/hybrid). With a Hybrid Deployment, your Dagster code is executed in your infrastructure. Refer to the [Getting started with Serverless Deployment guide](/dagster-cloud/getting-started/getting-started-with-serverless-deployment) if you want to use Serverless.
+
+We'll walk you through setting up your Dagster Cloud account, setting up an agent, loading your code, launching your first run, and configuring continuous integration.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need:
+
+- **Access to some Dagster code.** You can either use your own code or a [project scaffold](/getting-started/create-new-project). This guide uses a scaffold.
+
+- **To have `dagster` and `dagster-cloud` installed**. To install, run:
+
+  ```shell
+  pip install dagster dagster-cloud
+  ```
+
+- **The ability to create and configure an agent in your infrastructure.** Using a Hybrid Deployment requires setting up an agent for use with Dagster Cloud. The agent runs in your infrastructure, executes your code, and streams metadata about code execution over HTTPS back to Dagster Cloud.
+
+  Dagster Cloud supports a handful of agents, each with its own prerequisites for setup. Refer to the [Agent documentation](/dagster-cloud/deployment/agents) for more info.
+
+---
+
+## Step 1: Create a Dagster Cloud account and organization
+
+If you don't already have a Dagster Cloud account, [sign up for one](https://dagster.cloud/) before continuing.
+
+You'll also be asked to create a Dagster Cloud organization. When your organization is created, a single deployment, named `prod`, will also be created. Deployments are standalone environments, allowing you to operate independent instances of Dagster with separately managed permissions.
+
+---
+
+## Step 2: Select Hybrid as your deployment type
+
+1. When prompted, click **Select** under **Hybrid deployment**.
+2. The modal that displays will contain a pre-generated [agent token](/dagster-cloud/account/managing-user-agent-tokens) and info about the agents currently supported by Dagster Cloud.
+3. Follow the steps for setting up and deploying your agent:
+
+   - [Amazon Elastic Container Service (ECS)](/dagster-cloud/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc)
+   - [Docker](/dagster-cloud/deployment/agents/docker)
+   - [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+   - [Local](/dagster-cloud/deployment/agents/local)
+
+Continue to the next step when finished.
+
+---
+
+## Step 3: Deploy your code
```

Comment:
@yuhan - Would you mind giving this section another look? I made some changes based on comments from you and Pete, but I'm not totally confident this is fully correct.

Reply:
scaffolding piece lg2m